client.py

```diff
@@ -0,0 +1,197 @@
+""" Shest Gui Client """
+
+
+from CONSTANTS import DOWNLOAD_PATH, UPLOAD_PATH, REGULAR_PATH, \
+    DOWNLOAD_COMMAND, UPLOAD_COMMAND, KEY
+from cryptography.fernet import Fernet
+from pathlib import Path
+from re import search
+import requests
+import time
+import tkinter
+from typing import List
+
+
+def window_configuration() -> None:
+    """ Configure the main client tkinter window """
+
+    root = tkinter.Tk()
+    frame = tkinter.Frame(root)
+    command_inp = tkinter.Text(frame, width=70, height=1, fg="green")
+    frame.grid(row=0, column=0)
+    title = tkinter.Label(frame, text="Hy there! Please type a command and I "
+                                      "will execute it for you:\nupload: upload"
+                                      " <path>\ndownload: download <path>",
+                          font=(None, 15))
+    title.grid(row=0, column=0)
+    space1 = tkinter.Label(frame, text="\n")
+    space1.grid(row=1, column=0)
+    command_inp.grid(row=2, column=0)
+    confirm = tkinter.Button(frame, text="execute",
+                             command=lambda: retrieve_input(command_inp, frame))
+    confirm.grid(row=2, column=1)
+    space2 = tkinter.Label(frame, text="\n")
+    space2.grid(row=3, column=0)
+    root.mainloop()
```

Comment:
You configure the window **and** start it in the same function. Return the window instance to `main` so that it call from there `mainloop`.